pkgs/build-support/setup-hooks/wrap-gapps-hook.sh

```diff
@@ -36,9 +36,12 @@ wrapGAppsHook() {
   done
 
   if [ -z "$dontWrapGApps" ]; then
-    for i in $prefix/bin/* $prefix/libexec/*; do
-      echo "Wrapping app $i"
-      wrapProgram "$i" "${gappsWrapperArgs[@]}"
+    # Loop over files, without falling prey to filenames containing
```

Comment:
I don't believe the problem this change attempts to fix actually exists:

~~~
$ touch "this is a test"
$ for n in *this*; do echo "$n"; done
this is a test
~~~